collection/lens.rkt

```diff
@@ -39,6 +67,21 @@
   (make-lens (λ (target) (nth target index))
              (λ (target value) (set-nth target index value))))
 
+(define (mapper-lens lens)
```

Comment:
Yes lenses are supposed to be nouns, but that's a reason to name them so that they sound more like nouns. `ref-lens` sounds more like a noun that `map-lens`, because the word reference is a noun.